packages.yml

```diff
@@ -1,5 +1,6 @@
 packages:
-# - local: ../dbt_recurly_source
+  # - package: fivetran/recurly_source
+  #   version: [">=0.1.0", "<0.2.0"]
```

Comment:
Obligatory request to switch before release!

Reply:
Indeed!